lib/versions.json

```diff
@@ -1,4 +1,11 @@
 {
+   "c03111": {
+      "relDate": "Q2 2019",
+      "model": "4 model B",
+      "PCBRev": "1.0",

```

Comment:
According to elinux.org, the PCBrev is 1.1.

If you fix that we can merge this in. Thanks for your PR!